### PR TITLE
Add new icons

### PR DESCRIPTION
**Pull request by jsonunger**
Noting here, the `plant` and `piggyBank` icons look wrong, so I will circle back with Scott/design on Monday to get those fixed, but figured they could be in the repo and I can just update the SVG in the GCS bucket.

- additionalPolicy: 
- baby: 
- hands: 
- lunchPail: 
- piggyBank: 
- plant: 
- replacePolicy:

### CODE REVIEW

File: src/atoms/Icon/constants.js [hunk 1]
```diff
@@ -163,6 +165,7 @@ module.exports = {
     'lock',
     'lockBlue',
     'lockOrange',
+    'lunchPail',
```
Comment (mttrms) [inline src/atoms/Icon/constants.js:168]: LOL I think this is a briefcase but I like lunchpail

**Review by mttrms — APPROVED** [review]
LGTM!

File: src/atoms/Icon/constants.js (hunk 1 above)
Comment (showell215) [reply src/atoms/Icon/constants.js:168]: we could call it `rectangleWithHandle` to keep it generic? 😂

Comment (jsonunger) [reply src/atoms/Icon/constants.js:168]: Screen Shot 2020-12-30 at 11 26 12 AM
I just used the name of the image in Figma

**Pull request merged**